vertx-infinispan/src/main/asciidoc/index.adoc

```diff
@@ -432,3 +432,101 @@ After creation, the health check can be exposed over HTTP with a link:../../vert
 ----
 {@link examples.Examples#healthCheckHandler(io.vertx.core.Vertx, io.vertx.ext.healthchecks.HealthChecks)}
 ----
+
+== Using a different Infinispan version
+
+Switching the version to Infinispan 14 or Infinispan 14 Jakarta requires Java 11.
+
+=== Switch to Infinispan 14
+
+In order to use Infinispan 14 you need to upgrade all Infinispan related packages.
+
+Maven:
+----
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.infinispan</groupId>
+        <artifactId>infinispan-bom</artifactId>
+        <version>14.0.14.Final</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+----
+
+Gradle:
+----
+configurations.all {
+    resolutionStrategy.eachDependency { details ->
+        if (details.requested.group == 'org.infinispan') {
+            details.useVersion '14.0.14.Final'
+        }
+    }
+}
+----
+
+=== Switch to Infinispan 14 Jakarta
+
+In order to use Infinispan 14 you need to upgrade all Infinispan related packages and replace some jakarta specific artifacts.
+
+Maven:
+----
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.infinispan</groupId>
+        <artifactId>infinispan-bom</artifactId>
+        <version>14.0.14.Final</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-core-jakarta</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-multimap-jakarta</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-infinispan</artifactId>
+      <exclusions>
+        <exclusion>
```

Comment:
`infinispan-commons` should be excluded as well.

Perhaps others, depending on your findings when run the Maven dependency tree goal

Reply:
It turned out that vertx-web-sstore-infinispan never had the profiles, so there was no jakarta version of it .. I added the profiles and checked the dependency tree.

Reply:
done